helpers.py

```diff
@@ -81,14 +109,135 @@ def create_ssh_key(driver):
     ssh_key_remove_field.click()
     # Confirm deletion
     alert = driver.switch_to_alert()
     alert.accept()
+    time.sleep(5)
+
+def create_resource(driver, project_name, resource_name, time_wait):
+    dashboard_field = driver.find_element_by_link_text('Dashboard')
+    dashboard_field.click()
+    time.sleep(5)
+    projects = driver.find_element_by_link_text(project_name)
+    projects.click()
+    # organization = driver.find_element_by_link_text('Manage projects')
+    # organization.click()
+    # time.sleep(5)
+    # project_search_field = driver.find_element_by_css_selector('[ng-model="entityList.searchInput"]')
+    # project_search_field.send_keys(project_name)
+    # time.sleep(5)
+    # project = driver.find_element_by_link_text(project_name)
+    # project.click()
+    time.sleep(5)
+    resource_vms = driver.find_element_by_css_selector('[visible="vms"]')
+    resource_vms.click()
+    time.sleep(5)
+    resource_vms_creation = driver.find_element_by_link_text('Create')
+    resource_vms_creation.click()
+    time.sleep(5)
+    category = driver.find_element_by_class_name('desktop')
+    category.click()
+    time.sleep(5)
+    provider = driver.find_element_by_class_name('openstack')
+    provider.click()
+    time.sleep(5)
+    resource_field = driver.find_element_by_css_selector('[ng-model="AppStore.instance[field.name]"]')
+    resource_field.send_keys(resource_name)
+    time.sleep(5)
+    images = driver.find_elements_by_class_name('appstore-template-image')
+    for image in images:
+        if image.text == 'Ubuntu 14.04 x86_64':
+            image.click()
+            break
+    time.sleep(5)        
+    flavors = driver.find_elements_by_class_name('title')
+    for flavor in flavors:
+        if flavor.text == 'm1.small':
+            flavor.click()
+            break
+    time.sleep(5)
+    public_keys = driver.find_elements_by_class_name('description-ssh_public_key')
+    for public_key in public_keys:
+        if public_key.text == 'Openstack test key':
+            public_key.click()
+            break  
+    time.sleep(5)
+    purchase = driver.find_element_by_css_selector('[ng-click="AppStore.save()"]')
+    purchase.click()
+    time.sleep(time_wait)
+    driver.refresh()
+
+def delete_resource(driver, resource_name, project_name, time_wait_again, time_wait_remove):
+    dashboard_field = driver.find_element_by_link_text('Dashboard')
+    dashboard_field.click()
+    time.sleep(5)
+    projects = driver.find_element_by_link_text(project_name)
+    projects.click()
+    time.sleep(5)
+    vms = driver.find_element_by_css_selector('[visible="vms"]')
+    vms.click()
+    time.sleep(5)
+    resource_field = driver.find_element_by_css_selector('[ng-model="generalSearch"]')
+    resource_field.send_keys(resource_name)
+    time.sleep(5)
+    resource_list = driver.find_elements_by_class_name('list-box')
+    for resource in resource_list:
+        assert 'Online' in resource.text, 'Warning'
```

Comment:
`Warning` is meaningless. `Error: cannot delete resource that is not online or does not exist`